src/machine/machine_avr.go

```diff
@@ -154,3 +154,6 @@ var (
 	// UART0 is the hardware serial port on the AVR.
 	UART0 = UART{Buffer: NewRingBuffer()}
 )
+
+// SPI on AVR.
+var SPI0 = SPI{}
```

Comment:
This should be in the same file as where the SPI is defined, as SPI on AVR depends on fixed pin numbers.